do not error out when no DM is found. That fine , one may want to install just base system

diff -Naur 3.2.2-187/src/modules/displaymanager/main.py 3.2.2-187-p/src/modules/displaymanager/main.py
--- 3.2.2-187/src/modules/displaymanager/main.py	2019-01-02 17:45:42.000000000 +0100
+++ 3.2.2-187-p/src/modules/displaymanager/main.py	2019-01-02 22:49:15.887832111 +0100
@@ -735,7 +735,7 @@
     if libcalamares.globalstorage.contains("displayManagers"):
         displaymanagers = libcalamares.globalstorage.value("displayManagers")
 
-    if not displaymanagers:
+    if displaymanagers is None:
         return (
             _("No display managers selected for the displaymanager module."),
             _("The displaymanagers list is empty or undefined in both"
@@ -767,7 +767,7 @@
             if dm in displaymanagers:
                 displaymanagers.remove(dm)
 
-    if not dm_impl:
+    if dm_impl is None:
         return (
             _("No display managers selected for the displaymanager module."),
             _("The list is empty after checking for installed display managers.")
